### PR TITLE
Added migration to update fastn_user identity can be null

### PR DESCRIPTION
**Pull request by Arpita-Jaiswal**

### CODE REVIEW

File: ft-sdk/migrations/0003-update-fastn-user-identity.sql
Comment (amitu) [inline ft-sdk/migrations/0003-update-fastn-user-identity.sql:1]: Rename this to `0003-make-fastn-user-identity-nullable.sql`.

Comment (Arpita-Jaiswal) [reply ft-sdk/migrations/0003-update-fastn-user-identity.sql:1]: Done

**Comment by amitu** [comment]
@Arpita-Jaiswal update the changelog also.

File: ft-sdk/migrations/0003-fastn-user-identity-nullable-and-fastn-session-data-non-nullable.sql [hunk 1]
```diff
@@ -58,5 +58,5 @@ ALTER TABLE "new__fastn_session" RENAME TO fastn_session;
 
 
 
--- Step 6: Re-enable foreign key constraint enforcement (if necessary)
+-- Re-enable foreign key constraint enforcement (if necessary)
 PRAGMA defer_foreign_keys = OFF;
```
Comment (amitu) [inline ft-sdk/migrations/0003-fastn-user-identity-nullable-and-fastn-session-data-non-nullable.sql:62]: You do not have to do this, as it automatically is set to off at the end of transaction.

**Pull request merged**